src/app/chat/chat.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="mb-32">
+<div class="mt-24">
```

Comment:
you have to keep `mb-32`. Otherwise, conversation will get under to message input component when the conversation is too long.

Reply:
The thing is when i made the navbar sticky the Chat content was going under the navbar, can you check once when you are available, this may be due to position absolute internally being applied in which the content is disturbed and the Chat content is not aware that navbar is actually present there.

Reply:
I mean you have to use both :)

`class="mb-32 mt-24"`